edgelet/edgelet-docker/src/settings.rs

```diff
@@ -33,6 +34,7 @@ pub struct MobyRuntime {
     #[serde(with = "url_serde")]
     uri: Url,
     network: MobyNetwork,
+    content_trust_root_ca_registry: Option<HashMap<String, PathBuf>>,
```

Comment:
Just to confirm, the string is the GUN, right?

Reply:
Its the registry name URL. ex. myregistry.azurecr.io
GUN would be myregistry.azurecr.io/myimage 

we get the image name and its tag via deployment manifest - we would use that for notary lookup